DataFormats/CTPPSReco/interface/CTPPSLocalTrackLite.h

```diff
@@ -0,0 +1,72 @@
+/****************************************************************************
+*
+* This is a part of TOTEM offline software.
+* Authors:
+*   Jan Kašpar (jan.kaspar@gmail.com)
+*
+****************************************************************************/
+
+#ifndef DataFormats_CTPPSReco_CTPPSLocalTrackLite
+#define DataFormats_CTPPSReco_CTPPSLocalTrackLite
+
+/**
+ *\brief Local (=single RP) track with essential information only.
+ **/
+struct CTPPSLocalTrackLite
+{
+  public:
+    CTPPSLocalTrackLite(uint32_t pid=0, float px=0., float pxu=-1., float py=0., float pyu=-1., float pt=0., float ptu=-1.)
```

Comment:
please define a proper default constructor (without arguments).
Are there use cases for constructors where the first 3-4 arguments are not known while the last few can have default values?

Switch to class from struct